sched/scheduler/jobRunner.go

```diff
@@ -62,7 +61,7 @@ func runJob(job sched.Job, saga *s.Saga, nodes []cm.Node) {
 
 // Logic to execute a single task in a Job.  Ensures Messages are logged to SagaLog
 // Logs StartTask, executes Tasks, Logs EndTask
-func runTask(saga *s.Saga, node cm.Node, taskId string, task sched.TaskDefinition) {
+func runTask(saga *s.Saga, worker worker.WorkerController, taskId string, task sched.TaskDefinition) {
```

Comment:
So I dislike this coupling.  Having the Interface which was just send Message allows us to implement any Node.  Not just Workers